translations/de.json/de.json

```
@@ -0,0 +1,129 @@
+	"Documentation complète": "Vollständige Dokumentation",
+	"Article de blog": "Blog-Post",
+	"L'ensemble des données produites avant, pendant et après le projet du mois sont disponibles aux adresses suivantes ;": "Alle vor, während und nach dem Projekt des Monats erstellten Daten sind unter den folgenden Adressen verfügbar:",
+	"Retrouvez les précédents projets du mois, leurs outils, statistiques et données.": "Finden Sie die vorherigen Projekte des Monats, ihre Tools, Statistiken und Daten.",
+	"En savoir plus": "Mehr erfahren",
+	"Code source du projet": "Quellcode des Projekts",
+	"Projets précédents": "Frühere Projekte",
+	"Précédent": "Zurück",
+	"Suivant": "Nächste",
+	"Afficher/masquer la couche": "Layer anzeigen/ausblenden",
+	"Télécharger un fichier KML pour contribuer avec une application de smartphone": "Laden Sie eine KML-Datei hoch, um mit einer Smartphone-Anwendung beitragen",
+	"Exporter les points à vérifier": "Die zu prüfenden Punkte exportieren",
+	"Objets à cet endroit": "Objekte an dieser Stelle",
+	"Modifier la position": "Lage ändern",
+	"Oui": "Ja",
+	"Non": "Nein",
+	"Ne sais pas": "Weiss nicht.",
+	"Envoyer les informations sur OpenStreetMap": "Daten nach OpenStreetMap hochladen",
+	"Retourner à la carte sans envoyer à OpenStreetMap": "Zurück zur Karte, ohne Update an OpenStreetMap zu senden",
+	"Annuler": "Annullieren",
+	"La création de notes de carte est à réserver aux cas ambigus, si la situation est claire préférez un autre mode d'édition.": "Die Erstellung von Kartenhinweisen ist für mehrdeutige Fälle reserviert. Wenn die Situation klar ist, bevorzugen Sie einen anderen Bearbeitungsmodus.",
+	"Commentaire": "Kommentar",
+	"Envoyer le commentaire de carte sur OpenStreetMap": "Kommentar zur Karte auf OpenStreetMap einschicken",
+	"Créer": "Erstellen",
+	"Envoi en cours, veuillez patienter": "Upload in Bearbeitung, bitte warten",
+	"Votre contribution a bien été prise en compte, merci pour votre participation !": "Ihr Beitrag wurde berücksichtigt, vielen Dank für Ihren Beitrag!",
+	"Retour à la carte": "Zurück zur Karte",
+	"OK": "OK",
+	"Vous êtes connecté ; ce formulaire va éditer directement OpenStreetMap": "Sie sind eingeloggt; dieses Formular wird direkt OpenStreetMap bearbeiten",
+	"Vous n'êtes pas connecté ; ce formulaire va créer une note dans OpenStreetMap": "Sie sind nicht eingeloggt; Dieses Formular wird einen Hinweis in OpenStreetMap erstellen",
+	"Si vous quittez la page, les modifications en cours seront perdues.": "Wenn Sie die Seite verlassen, gehen die aktuellen Änderungen verloren.",
+	"Valider": "Validieren",
+	"Des données sont manquantes, merci de réessayer": "Daten fehlen, bitte versuchen Sie es noch einmal.",
+	"Une erreur s'est produite, merci de réessayer plus tard": "Ein Fehler passierte, bitte versuchen Sie es später noch einmal.",
+	"Objet concerné ;": "Betroffenes Objekt;",
```

That should be colons, rather than semicolons.
```suggestion
	"Objet concerné :": "Betroffenes Objekt:",
```